File: 2019-3-12/Test/Test/ViewController.m
//
//  ViewController.m
//  Test
//
//  Created by 吉腾蛟 on 2019/3/13.
//  Copyright © 2019 mj. All rights reserved.
//

#import "ViewController.h"
#import "PreviewImage.h"

@interface ViewController ()
@property (weak, nonatomic) IBOutlet UIImageView *imageView1;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    [PreviewImage showBigImageWithTarget:self.imageView1];
}


@end

File: 2019-01-05/JustForFun/JustForFun/ViewController.swift
//
//  ViewController.swift
//  JustForFun
//
//  Created by admin on 2019/1/5.
//  Copyright © 2019 Sony. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    @IBOutlet weak var textLabel: UILabel!
    @IBAction func showAlert(_ sender: UIButton) {
        let alert=UIAlertController(title: "Hello", message: "Hello,world!", preferredStyle: UIAlertController.Style.alert)
        alert.addAction(UIAlertAction(title: "Close", style: UIAlertAction.Style.default, handler: nil))
        self.present(alert, animated: true, completion: nil)
        self.textLabel.text="点击了"
        
    }
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }


}

